docs/labs/shell-injection.html

```diff
@@ -0,0 +1,162 @@
+<!DOCTYPE html>
```

Comment:
This `shell-injection.html` file is identical to what's already merged into the `main` branch, so I have no comments here.

Reply:
I'm not sure why this file is part of the PR, since it's already in the main branch.  But it's not a serious problem, git can deal with unchanged files :-).

Reply:
Please do not *DELETE* the file from the branch at this point. That would tell git you want to remove this file, which is *not* what you want :-).